### PR TITLE
added: small text component

### PR DESCRIPTION
**Pull request by som749898**
small text component has been adde
Screenshot (90)
d

### CODE REVIEW

**Comment by Digambar1411** [comment]
Can be merged

**Review by Digambar1411 — APPROVED** [review]

**Review by sakshi006 — APPROVED** [review]
Looks good to merge

**Comment by srinibas39** [comment]
You can add little margin to your texts ,otherwise looks good to merge.

File: Components/Text-Utilities/text.html [hunk 1]
```diff
@@ -52,21 +52,31 @@ <h2 class="heading2 py-bottom-1x">Text Utilities</h2>
...
-          &ltdiv class="heading3">Normal Text&lt/div>
-          &ltdiv class="heading4">Small Text&lt/div>
+          &ltdiv class="heading1">This is an h1 heading&lt/div>
+          &ltdiv class="heading2">This is an h2 heading&lt/div>
+          &ltdiv class="heading3">This is an h3 heading&lt/div>
+          &ltdiv class="heading4">This is an h4 heading&lt/div>
+        </code></pre>
+        <h3 class="heading3 py-bottom-1x">Small Text</h3>
+        <p class="text-size py-bottom-1x">For small text you can use the class names txt-small and txt-xs.</p>
+        <div class="background-cover flex flex-column vertical-center horizontal-center mg-bottom-1x py-1x">
```
Comment (anupkgurung) [inline Components/Text-Utilities/text.html:69]: what is 1x in py-1x
try to use meaning full class names.

**Review by anupkgurung — COMMENTED** [review]
class name can have a bit more specific or purfull names.
rest seems good.

**Pull request merged**